src/lightly_train/_methods/dinov2/dinov2.py

```diff
@@ -230,6 +230,8 @@ def __init__(
         freeze_eval_module(self.teacher_head)
 
         # Losses
+        # TODO(Jonas 06/25): make two loss versions one for centering softmax and one for sinkhorn knopp, 
+        # so we could instantiate the corresponding one and remove logic form the train loop
```

Comment:
The way we solved this in LightlySSL is to make a Center module that takes a center mode as input: https://github.com/lightly-ai/lightly/blob/90ca6abf4cbd34df6e0b58f675d92dc194883602/lightly/models/modules/center.py#L1